### PR TITLE
Wind

### LINKED ISSUE #4330: Wind does not update without windgens
Noticeable now that there is a widget for it.  

It should be at least updated on frame 0 (players can force an update by nanoframing a windgen), or more likely in pregame so that you can plan your buildorder around it?

### PR DESCRIPTION
**Pull request by finestep**
Fixes #4330
Pregame metaphysics are unknown territory to me

### CODE REVIEW

File: LuaRules/Gadgets/unit_windmill_control.lua [hunk 1]
```diff
@@ -109,22 +109,26 @@ end
 -------------------------------------------------------------------------------------
 -------------------------------------------------------------------------------------
 
+local function scaleWind(str) 
+	return (strength - windMin)/windRange
```
Comment (sprunk) [inline LuaRules/Gadgets/unit_windmill_control.lua:113]:
```suggestion
local function scaleWind(str) 
	return (str - windMin)/windRange
```

**Comment by GoogleFrog** [comment]
Does the value in Initialize actually mean anything? Eg it is a bit misleading to report a number pregame only for it to change instantly.

**Comment by finestep** [comment]
As far as I can tell, there should be no discontinuity from Initialize() to GameFrame(0) in wind strength.
Regardless Initialize() does not seem to be the appropriate place to do this since wind widget still shows 0% in pregame for me

**Comment by GoogleFrog** [comment]
I doubt resource bar does any updates prior to frame 0. To initialise the wind display correctly you would need to add a display update to something like widget:Initialize.

**Pull request closed without merging**